FormattingStyleGuide.md

```diff
@@ -0,0 +1,43 @@
+# Formatting & Style Guide
+
+## Scope and purpose
+This document provides formatting and style conventions for publishing documents in __GitHub__ in __Markdown__ format. Some advice on document conversion and how to meet more sophisticated publishing needs is offered.
+
+## General
+- Use [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/).
+- If the workflow prescribes the document to go into BitBucket repository first, then you may find [BitBucket Markdown Syntax Guide](https://confluence.atlassian.com/bitbucketserver/markdown-syntax-guide-776639995.html) useful.
+
+## Title & headings
+- Use # (h1) for document title.
+- Use ## (h2) for top-level headings.
+- Number the headings manually. There's no heading numbering support in Markdown.
+
+## Table of contents
+- Provide table of contents for longer documents.
+- Use links in the table of contents. Use [GitHub heading anchors](https://gist.github.com/asabaylus/3071099). For example,
+```
+  [Table of contents](StyleManual.md#table-of-contents) 
+```
+links to this section.
+
+## Linking
+- Use [relative linking)(https://github.com/blog/1395-relative-links-in-markup-files) to link to image files and other documents in the same repository.
+
+## Code
+- Use backticks to mark inline code, URLs a.o. constant values.
+- Use GitHub Markdown code fencing to format blocks of code.
+
+## Tables
+- Use GitHub Flavored Markdown features.
```

Comment:
X-Road documentation contains a lot of tables. These are very challenging. Tables are so large that they do not look good as raw MD on a text editor. They are difficult to read and also a little difficult to edit. On the other hand, using a good MD editor with preview, the tables look good when. Also github renders them nicely.

Some tables contain multi line columns that is not supported by githug flavored markdown. The solution I found is to use <br/> tags between lines and the tables render correctly, but again, do not look good as raw text.